python/cudf/cudf/_lib/includes/copying.pxd

```diff
@@ -5,29 +5,31 @@
 # cython: embedsignature = True
 # cython: language_level = 3
 
-from cudf.bindings.cudf_cpp cimport *
+from cudf._lib.cudf cimport *
 
 
 cdef extern from "cudf/copying.hpp" namespace "cudf" nogil:
 
-    cdef void gather(
-        const cudf_table * source_table,
+    void gather(
+        cudf_table * source_table,
         const gdf_index_type* gather_map,
         cudf_table* destination_table
     ) except +
```

Comment:
Why did we remove the cdef / const here?

Reply:
My bad - will put them back in.